app/jobs/import_versions_job.rb

```diff
@@ -69,6 +71,12 @@ def import_record(record)
       return
     end
 
+    if page.uuid_previously_changed?
+      log(object: page, operation: :created, row: row)
```

Comment:
How come this isn’t being done down in `page_for_record`? I had to spend a minute puzzling out why you were checking `uuid_previously_changed?` (it was non-obvious to me that what’s really happening is that the uuid would only have changed if the object was new), plus it seems prone to potential breakage if `page_for_record` changes.

Reply:
My original intention was to keep all of the logging in the `#import_record` method, both for conceptual reasons  and also so I didn't have to pass down the `row` variable into every other `#*_for_record` method.  ...though I think in practice the original concept wasn't practical once I integrated your last round of feedback.

I'll drop it down into the extracted `page_for_record`.